server/channels/app/notification_push.go

```diff
@@ -148,7 +148,7 @@ func (a *App) sendPushNotificationToAllSessions(rctx request.CTX, msg *model.Pus
 	for _, session := range sessions {
 		// Don't send notifications to this session if it's expired or we want to skip it
 		if session.IsExpired() || (skipSessionId != "" && skipSessionId == session.Id) {
-			a.CountNotificationReason(model.NotificationStatusNotSent, model.NotificationTypePush, model.NotificationReasonSessionExpired)
+			a.CountNotificationReason(model.NotificationStatusNotSent, model.NotificationTypePush, model.NotificationReasonSessionExpired, "")
```

Comment:
This seems like another place that we might have the user's platform

Reply:
Same as the previous comment: only dependent on server code, and we haven't yet extracted the platform.

Reply:
I think it makes sense to add platform here, would it be an issue if we extract the platform earlier?

Reply:
We can, but what information do we get from that?

Expired sessions are platform independent. Knowing from which device the session is expired tell us nothing really.